packages/mobile/src/utils/LogUploader.ts

```diff
@@ -49,7 +49,7 @@ export default class FirebaseLogUploader {
     await RNFS.moveFile(logFilePath, tmpFilePath)
     // Don't "await" here. While everything else in this async method is relatively fast, data upload can be slow
     // and we don't want to block the user from using the app.
-    return FirebaseLogUploader.uploadLogsToFirebaseStorage(tmpFilePath, uploadPath, uploadFileName)
```

Comment:
This `return` was functionally an `await`. Given the comment above, this feels like the right one to get rid of.

Reply:
So should we remove the comment?